src/app-layout/drawer/interfaces.ts

```diff
@@ -39,22 +39,22 @@ export interface DesktopDrawerProps {
 }
 
 export interface ResizableDrawerProps extends DesktopDrawerProps {
-  activeDrawer?: PublicDrawer;
+  activeDrawer: AppLayoutProps.Drawer | undefined;
   onResize: (resizeDetail: { size: number; id: string }) => void;
   size: number;
   getMaxWidth: () => number;
   refs: FocusControlRefs;
-  toolsContent?: React.ReactNode;
+  toolsContent: React.ReactNode;
```

Comment:
can this one be undefined?

Reply:
`undefined` is embedded inside `ReactNode` typings.

`?` is a problem, which I in the previous PR: https://github.com/cloudscape-design/components/pull/1697#discussion_r1376059808

I am trying to gradually remove `?` from our internal components and recommend everybody to do the same